core/services/feeds/service.go

```diff
@@ -607,6 +635,15 @@ func (s *service) ProposeJob(ctx context.Context, args *ProposeJobArgs) (int64,
 	return id, nil
 }
 
+func isWFSpec(spec string) bool {
```

Comment:
looks like generateJob() calls job.ValidateSpec(spec) which gives it jobType - can we do the same here?

Reply:
ah, good idea, let me check